src/simulator_core/entities/assets/pipe.py

```diff
@@ -149,11 +149,10 @@ def add_physical_data(self, esdl_asset: EsdlAssetObject) -> None:
         """
         # Error handling is performed in EsdlAssetObject.get_asset_parameters
         self.length, _ = esdl_asset.get_property(
-            esdl_property_name="length", default_value=self.length
-        )
+            esdl_property_name="length", default_value=self.length)
         self.roughness, _ = esdl_asset.get_property(
-            esdl_property_name="roughness", default_value=self.length
-        )
+            esdl_property_name="roughness", default_value=self.roughness)
+        self.roughness = PIPE_DEFAULTS.k_value if self.roughness == 0 else self.roughness
```

Comment:
We moeten nog een keer kijken of dit de beste manier is. Ik zou zelf zeggen dat dit bijvoorbeeld in get_property moet zitten? Of was er een reden waarom we het zo hadden gedana.

Reply:
Zou inderdaad beter zijn, maar als je aan ESDL vraagt krijg je 0 terug als er geen waarde is dus daar moet je op checken

Reply:
Ja de methode checkt op missing values maar dus niet op 0. In principe kan dit ook een oplossing zijn, want anders stop je die "slimmigheid" in die andere methode